pzsd_bot/cogs/points.py

```diff
@@ -53,6 +60,13 @@ def page_buttons(self) -> list[PaginatorButton]:
             PaginatorButton("last", label=">>", style=discord.ButtonStyle.blurple),
         ]
 
+    def validate_name(self, name: str) -> Enum:
+        if re.fullmatch(r"(?:every|no)[ -]?(?:one|body)|me", name):
```

Comment:
are all of these possibilities really all reserved names? "every body" "no one" "no-one" etc? I would think there should be a discrete set of reserved names that are listed out

Reply:
That's the idea. I suppose I could switch to discretely listing out every name and all its variations, but I just felt like the actual amount of names I don't want people to use was pretty small, and their variations rather large.

Reply:
why are all these permutations even reserved? I only see this one hard coded reference here https://github.com/fiskenslakt/pzsd-bot/blob/7818e5f71f3657b57f9041415196e76cabb5475c/pzsd_bot/cogs/points.py#L104

Reply:
actually there are a few hard coded references to "everyone" but none of the other permutations

Reply:
you should just make "everyone" into a constant somewhere and reference that